src/current/v24.1/plpgsql.md

```diff
@@ -409,6 +478,111 @@ BEGIN
   END
 ~~~
 
+### Perform transaction control
+
+Use a `COMMIT` or `ROLLBACK` statement to issue a group of PL/pgSQL statements as an explicit transaction within a [stored procedure]({% link {{ page.version.version }}/stored-procedures.md %}). Any updates made by those statements are either committed or rolled back, while [PL/pgSQL variables](#declare-a-variable) keep their values. 
+
+~~~ sql
+BEGIN
+	statements
+	[ COMMIT | ROLLBACK ]
+~~~
+
+`COMMIT` and `ROLLBACK` statements within PL/pgSQL blocks have the following requirements:
+
+- They must be used inside a PL/pgSQL [stored procedure]({% link {{ page.version.version }}/stored-procedures.md %}).
+{% comment %}- The procedure must be called directly in a `CALL` statement, or all of its ancestors must be stored procedures or `DO` blocks{% endcomment %}.
```

Comment:
I think we should remove this point in favor of the next one, since it's more strict.

Reply:
This is actually hidden due to the comment tags - I'm just leaving it here so that it's easier to document in the future.